### PR TITLE
ember-cli notes 27 July 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-07/july-27.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
+
+
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-07/july-27.md]: Please reply here with any agenda items

Comment (Turbo87) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/ember-cli/pull/7207

Comment (Turbo87) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/ember-cli/pull/7137 /cc @kellyselden

Comment (Turbo87) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/ember-cli/pull/6661

Comment (Turbo87) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/ember-cli/issues/6451

Comment (rwjblue) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/rfcs/pull/109

Comment (kratiahuja) [inline ember-cli/2017-07/july-27.md]: https://github.com/ember-cli/ember-cli/pull/7245

Comment (rwjblue) [inline ember-cli/2017-07/july-27.md]: I would also like to discuss using native `Symbol` for private methods/properties going forward.  Mostly as a mitigation to an addon knowingly reaching in and modifying underscored properties/methods, and us eventually being worried about breaking the use case that the addon solves...

Comment (Turbo87) [inline ember-cli/2017-07/july-27.md]: on behalf of @cibernox: https://github.com/ember-fastboot/ember-cli-fastboot/pull/457

**Comment by hjdivad** [comment]
cc @rwjblue @twokul @cibernox @hjdivad @bcardarella @kellyselden @turbo87 @kratiahuja

**Pull request merged**